### PR TITLE
added basic Docker support

### PR DESCRIPTION
**Pull request by fmhc**
Hello, I added some basic nginx / docker support, please review :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,4 +107,4 @@ If you plan to use real money with this, USE AT YOUR OWN RISK.
 BTC [3PK1bBK8sG3zAjPBPD7g3PL14Ndux3zWEz](bitcoin:3PK1bBK8sG3zAjPBPD7g3PL14Ndux3zWEz)<br>
 XMR 48NJj3RJDo33zMLaudQDdM8G6MfPrQbpeZU2YnRN2Ep6hbKyYRrS2ZSdiAKpkUXBcjD2pKiPqXtQmSZjZM7fC6YT6CMmoX6<br>
 COINBASE
-https://commerce.coinbase.com/checkout/c58bd003-5e47-4cfb-ae25-5292f0a0e1e8
\ No newline at end of file
+https://commerce.coinbase.com/checkout/c58bd003-5e47-4cfb-ae25-5292f0a0e1e8
```
Comment (fmhc) [inline README.md:110]: I added a blank last line i guess, did not touch the link, I dont get why this is marked as a change. pls review / check!

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+COPY package*.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+
+# production stage
+FROM nginx:stable-alpine as production-stage
+COPY --from=build-stage /app/dist /usr/share/nginx/html
+COPY nginx.conf /temp/nginx.conf 
+RUN envsubst / < /temp/nginx.conf > /etc/nginx/conf.d/default.conf
```
Comment (fmhc) [inline Dockerfile:14]: I don't know if this line is necessary, ran into some problems and copy-pasted half a line as a solution

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# build stage
+FROM node:lts-alpine as build-stage
+WORKDIR '/app'
+RUN apk --no-cache add git
```
Comment (fmhc) [inline Dockerfile:4]: without git, the npm install crashes as it tries to download some charting libs from git

File: nginx.conf [hunk 4]
```diff
@@ -0,0 +1,10 @@
+server {
+  listen 80;
+  server_name _ default_server;
+
+  root /usr/share/nginx/html;
+
+  location / {
+    try_files $uri $uri/ /index.html;
```
Comment (fmhc) [inline nginx.conf:8]: this redirects everything to the index.html so that the /abc123 links are working with the vue router.

**Review by fmhc — COMMENTED** [review]
Hey there, i added some docker support, please review and merge if you like :)

**Comment by Tucsky** [comment]
many tanks 🎉

**Pull request merged**